go.mod

```diff
@@ -1,7 +1,12 @@
 module github.com/nats-io/nats.go
 
 require (
+	github.com/golang/protobuf v1.4.0
 	github.com/nats-io/jwt v0.3.2
+	github.com/nats-io/nats-server/v2 v2.1.6
```

Comment:
I know that go mod tooling added this, but should we continue to remove it to avoid depending on server?

Reply:
Arg.. it's my local update of staticcheck that did that likely and I forgot to remove. Will update.